build/distributions/CentOS/6/x86_64/DVD/isolinux/boot.msg

```diff
@@ -7,7 +7,7 @@
     - This will erase your existing system!
     - You need at least 50GB of disk space for this to succeed.
     - Partitions will be configured to be STIG compliant.
-    - Add the option 0asimp_disk_crypt07 to encrypt your disk.
+    - Type 0asimp-nocrypt07 to disable disk encryption.
```

Comment:
I think this description is confusing.  I would rather see a description that started with "To auto-install SIMP, you can type \<ENTER\>, simp \<ENTER\>, or simp-nocrypt \<ENTER\>" and bullets that explain that \<ENTER\> and simp \<ENTER\> are identical and provide disk encryption, whereas simp-nocrypt \<ENTER\> does not.

Reply:
@lnemsick-simp If it gets much longer than this, it doesn't fit on the screen.

Reply:
@lnemsick-simp and @sharkbruhaha Open to rewording that doesn't span more than one line. An additional line will make the messages not fit on a standard terminal IIRC.